### DISCUSSION TITLE
Make a comunity version.

### DISCUSSION THREAD

**Discussion by ca3gamedev** [godotengine/godot-proposals#8950] [General Discussions]
Similar to how some IDEs have community versions, besides the basic edition.
Maybe godot could have an edition with mandatory assets, like dialogic, that would add basic core features, and are well integrated by default on the editor, so the user doesn't have to deal with installing the asset or having it exposed on an asset folder.

That way, addons like beehive could be better integrated and tested, more than it being an addon that lacks enough integration and polish in the editor by default.

**Comment by AThousandShips** [comment]
People are free to do this, and we already have a [list of add-ons](https://github.com/godotengine/awesome-godot), but providing a maintained version is a lot of work and back and forth keeping different add-ons up to date and managing if they fail to update and become incompatible, it's safer to leave this to users, or, again, a community effort as opposed to an official one

**Comment by ca3gamedev** [comment]
> People are free to do this, and we already have a [list of add-ons](https://github.com/godotengine/awesome-godot), but providing a maintained version is a lot of work and back and forth keeping different add-ons up to date and managing if they fail to update and become incompatible, it's safer to leave this to users, or, again, a community effort as opposed to an official one

That's the core idea, that the addon developers collaborate on a comunity edition, instead of having the core team be burdened with mantaining it.

**Reply by Calinou** [reply]
The godot-proposals repository is for things that are intended to be officially part of the engine, as opposed to efforts that are purely on the community's side.
